src/components/CallView/shared/VideoBackground.vue

```diff
@@ -21,7 +21,8 @@
 
 <template>
 	<div class="video-backgroundbackground">
-		<div class="darken">
+		<div class="darken"
+			ref="placeholder">
```

Comment:
can we improve the name? maybe "darkener"?

Reply:
darkener is not useful at all, as the thing isn't actually darker. Maybe `backgroundAvatar` ?

Reply:
It should darken, that should be the only purpose of that div